### DISCUSSION TITLE
诱导转向的伪开源战略 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#41] [Comments]
# 诱导转向的伪开源战略 | 夜天之书

本文已整理到【开源小镇】网站【企业沙盘】文档，最新更新请前往对应页面阅读。 本文是我继《企业实践开源的动机》、《开源世界当中到底存不存在“白嫖”？》和《免费增值的商业模式》之后再一次讨论免费增值相关的软件商业模式。主要受到 Percona 发布的 Open Source Bait and Switch: Licensing and Beyond 一文的启发，让我意识到 MongoDB Inc. 在

https://www.tisonkun.org/2022/10/04/bait-and-switch-fauxpen-source-strategy/

**Comment by mggger** [comment]
你好，能提供rss源吗？
